.github/workflows/unifiedBuildTestAndInstallStatic.yml

```diff
@@ -110,12 +117,19 @@ jobs:
         shell: bash
         run:
           echo enable_sccache="-DCMAKE_C_COMPILER_LAUNCHER=sccache -DCMAKE_CXX_COMPILER_LAUNCHER=sccache" >> $GITHUB_OUTPUT
+# Build static mimalloc
+      - name: Build microsoft/mimalloc
+        run: |
+          mkdir -p build/mimalloc/build
+          cd build/mimalloc/build
+          cmake ../ -G Ninja -DCMAKE_BUILD_TYPE=${{ inputs.cmake_build_type }} -DMI_OVERRIDE=ON -DCMAKE_C_COMPILER=clang-19 -DCMAKE_CXX_COMPILER=clang++-19
```

Comment:
`-DMI_OVERRIDE=ON` causes the binary to include `malloc` and `free` symbols. If built without this, then it will not create these. This is necessary if you want to have these win during linking.